client/my-sites/plans/ecommerce-trial/index.tsx

```diff
@@ -25,25 +32,46 @@ const ECommerceTrialPlansPage = ( props: ECommerceTrialPlansPageProps ) => {
 		} );
 	}, [] );
 
+	const goToCheckoutWithPlan = useGoToCheckoutWithPlan();
+
 	useOneDollarOfferTrack( siteId, 'plans' );
 
 	return (
 		<>
 			<BodySectionCssClass bodyClass={ [ 'is-ecommerce-trial-plan' ] } />
 
-			<div className="e-commerce-trial-plans__banner-wrapper">
-				<TrialBanner isEcommerceTrial={ true } />
+			<div
+				className={ classNames( 'e-commerce-trial-plans__banner-wrapper', {
+					'reduced-margin': isEntrepreneurTrial,
+				} ) }
+			>
+				<TrialBanner
+					isWooExpressTrial={ isWooExpressTrial }
+					isEntrepreneurTrial={ isEntrepreneurTrial }
+					callToAction={
+						! isWooExpressTrial ? (
+							<UpgradeButton
+								goToCheckoutWithPlan={ goToCheckoutWithPlan }
+								isEntrepreneurTrial={ ! isWooExpressTrial }
```

Comment:
Couldn't we just pass `isEntrepreneurTrial` here - since we already calculated it above? But maybe I am missing something. 🤔 
```suggestion
								isEntrepreneurTrial={ isEntrepreneurTrial }
```

Reply:
Thank you for pointing that. This condition was introduced before I created the isEntrepreneurTrial and I didn't remember to update that.